src/routes/bookmarks/+page.svelte

```diff
@@ -4,7 +4,7 @@
     import { BookmarkIcon } from '$lib/icons';
     import ShareIcon from '$lib/icons/ShareIcon.svelte';
     import Navbar from '$lib/components/Navbar.svelte';
-    import { refs, t } from '$lib/data/stores';
+    import { bookmarks, refs, t } from '$lib/data/stores';
```

Comment:
`bookmarks` was accidentally added.  Please remove.

Reply:
@mikaylahubbard I was able to change this through the website.  I would like to get this into the release that will be happening today or tomorrow